src/main/docker/dev/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM java:openjdk-8-jdk-alpine
+FROM openjdk:8-jre-alpine
```

Comment:
as the war is built locally and copy during the docker build, **jre** is enough here :)